pkg/kubelet/config/file_linux.go

```diff
@@ -80,7 +84,7 @@ func (s *sourceFile) watch() error {
 	}
```

Comment:
I did some fixes:
* add IN_ATTRIB for watching symlink
* introduce backoff for watching retry loop
* merge list & watch into one goroutine to get rid of lock